### PR TITLE
voting contract 0.10.0

### PR DESCRIPTION
**Pull request by levackt**
@ethanfrey Making a new PR after rebasing master, hope it's ok and I'll close #83 , let me know otherwise.

### CODE REVIEW

File: voting/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+This is a simple voting contract. It creates a contract to manage token weighted polls,
+where voters deposit native coins in order to vote.
+Voters can withdraw their stake, but not while a poll they've participated in is still in progress.
+
+Anyone can create a poll, and as the poll creator, only they are allowed to end/tally the poll.
+
+This contract is mainly considered as a simple tutorial example.
+
+As of v0.2.0, this was rebuilt from
```
Comment (ethanfrey) [inline voting/README.md:11]: Maybe you bump the number in Cargo.toml - I think it is only 0.1.0 - doesn't show how much hard work already went in.

File: voting/examples/schema.rs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InitMsg), &out_dir);
+    export_schema(&schema_for!(HandleMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+    export_schema(&schema_for!(State), &out_dir);
```
Comment (ethanfrey) [inline voting/examples/schema.rs:17]: You also want to add the other query responses: 

`PollResponse` and `TokenStakeResponse`

File: voting/src/state.rs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    pub yes_votes: Uint128,
+    pub no_votes: Uint128,
+    pub voters: Vec<CanonicalAddr>,
+    pub voter_info: Vec<Voter>,
+    pub end_height: u64,
+    pub start_height: Option<u64>,
+    pub description: String,
+}
+
+impl State {}
```
Comment (ethanfrey) [inline voting/src/state.rs:56]: Dead code, right?

File: voting/README.md (hunk 1 above)
Comment (levackt) [reply voting/README.md:11]: Thanks, bumped

File: voting/examples/schema.rs (hunk 2 above)
Comment (levackt) [reply voting/examples/schema.rs:18]: Good spot, will be handy on coralnet, checking it out now.

**Comment by levackt** [comment]
Big pleasure Ethan, cheers

**Pull request merged**